jvm/src/test/scala/org/scalacheck/GenSpecification.scala

```diff
@@ -289,6 +289,19 @@ object GenSpecification extends Properties("Gen") with GenSpecificationVersionSp
     }
   }
 
+  property("pick with gen") = forAll { (x: Int, y: Int, rest: List[Int]) =>
+    val lst = x :: y :: rest
+    forAll(choose(-1, 2 * lst.length)) { n =>
+      val gs = rest.map(Gen.const)
+      Try(pick(n, Gen.const(x), Gen.const(y), gs: _*)) match {
+        case Success(g) =>
+          forAll(g) { m => m.length == n && m.forall(lst.contains) }
```

Comment:
Does this assertion need to be `m.forall(x :: y :: lst.contains)`?

Reply:
That has a syntax error, but hopefully you catch my drift.

Reply:
No because `val lst = x :: y :: rest`.

Reply:
So it is.  I read `lst` as `rest`.